packages/cloud-cognitive/src/components/AddSelect/AddSelect.js

```diff
@@ -20,7 +20,6 @@ import {
 import { Tearsheet, TearsheetNarrow } from '../../components/Tearsheet';
 import { NoDataEmptyState } from '../../components/EmptyStates/NoDataEmptyState';
 import { pkg } from '../../settings';
-
 const componentName = 'AddSelect';
 
 export let AddSelect = forwardRef(
```

Comment:
A possible minor update

```suggestion
export const AddSelect = forwardRef(
```